src/mapping/config/MappingConfiguration.hpp

```diff
@@ -169,6 +178,11 @@ class MappingConfiguration : public xml::XMLTag::Listener {
   const std::string ATTR_SHAPE_PARAM    = "shape-parameter";
   const std::string ATTR_SUPPORT_RADIUS = "support-radius";
 
+  // Attributes for geometric multiscale
+  const std::string ATTR_GEOMULTISCALE_TYPE   = "multiscale-type";
+  const std::string ATTR_GEOMULTISCALE_AXIS   = "multiscale-axis";
+  const std::string ATTR_GEOMULTISCALE_RADIUS = "multiscale-radius";
```

Comment:
```suggestion
  const std::string ATTR_GEOMETRIC_MULTISCALE_TYPE   = "multiscale-type";
  const std::string ATTR_GEOMETRIC_MULTISCALE_AXIS   = "multiscale-axis";
  const std::string ATTR_GEOMETRIC_MULTISCALE_RADIUS = "multiscale-radius";
```

to be consistent with the type above

Reply:
Done 👍🏼